### PR TITLE
Site updates

### PR DESCRIPTION
**Pull request by GawainLynch**

### CODE REVIEW

File: view/source/scss/docs.scss [hunk 1]
```diff
@@ -864,7 +863,18 @@ div.gallery-popup img {
...
             background-color: #999;
         }
+    }
+
+    a.button.large.expand.docsintro {
+        flex-basis: 100%;
+    }
 
+    a.button.large.docsintro {
+        flex-basis: calc(50% - 0.5em);
```
Comment (SahAssar) [inline view/source/scss/docs.scss]: Could you change to 0.5em on `a.button.medium.docsintro` too?

Comment (GawainLynch) [inline view/source/scss/docs.scss]: Done and force pushed

Comment (GawainLynch) [inline view/source/scss/docs.scss]: See, this is why Gawain isn't allowed to do visual work :stuck_out_tongue_winking_eye:

**Comment by SahAssar** [comment]
:+1:

**Pull request merged**